doc/guides/bluetooth/mesh-pics.rst

```diff
@@ -0,0 +1,312 @@
+.. _mesh-pics:
+
+MESH ICS
+********
+
+PTS version: 7.4.1
+
+\* - different than PTS defaults
+
+\^ - field not available on PTS
+
+M - mandatory
+
+O - optional
+
+
+Major Profile Version (X.Y)
+===========================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_0_1	True		Mesh v1.0 (M)
+==============	==============	=======================================
+
+
+Minor Profile Version (X.Y.Z)
+=============================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_0A_1	True		Erratum 10395 (C.1)
+TSPC_MESH_0A_2	True		Mesh v1.0.1 (C.2)
+==============	==============	=======================================
+
+
+Bluetooth COre Specification v4.0 and Later
+===========================================
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_1_1	True		Core Speciciation 4.0 or later version (M)
+==============	==============	=======================================
+
+
+Roles
+=====
+
+==============	==============	=======================================
+Parameter Name	Selected	Description
+==============	==============	=======================================
+TSPC_MESH_2_1	True		Node (C.1)
+TSPC_MESH_2_2	False (*)	Provisioner (C.1)
+==============	==============	=======================================
+
+
+Node Capabilites - Bearers
```

Comment:
At least you were consistent in your misspelling: 10 occurrences of ``capabilites`` :), and update the title underlines too:

```suggestion
Node Capabilities - Bearers
```